cypress/e2e/helpers/surveyHelper.ts

```diff
@@ -46,14 +55,14 @@ export const SCHEDULE_TRIWEEKLY = (day: string, time: string) =>
 export const SCHEDULE_WEEKLY = (day: string, time: string) =>
   `Weekly on ${day} at ${time}`;
 
-export const verifySurveyCard = (
-  surveyName: string,
-  schedule: string,
-  status: string,
-) => {
+export const OPTION_RADIO_EXCELLENT = 'input[value="Excellent"]';
```

Comment:
Just a suggestion =)
```suggestion
export const RADIO_OPTION_EXCELLENT = 'input[value="Excellent"]';
```